### ISSUE TITLE
Boxes: give a way to omit the icon

### ISSUE THREAD

**Issue by damithc** [MarkBind/markbind#2559]
In some cases the user might want to omit the icon of a box.
![image](https://github.com/MarkBind/markbind/assets/1673303/352e4a82-a0a9-43ee-8098-aac22f92f37a)

Perhaps we can allow that in some way e.g., `icon=""`?

**Comment by tlylt** [comment]
Should the space occupied by the icon be retained (or removed, i.e. the paragraph should be aligned further left) if the icon is removed?

**Comment by Incogdino** [comment]
Hi may I have details on how to reproduce this behaviour? It seems that it is already part of a feature that is implemented by specifying `no-icon` as an option in `box`

**Comment by damithc** [comment]
> Hi may I have details on how to reproduce this behaviour? It seems that it is already part of a feature that is implemented by specifying `no-icon` as an option in `box`

Yup. Another case of 'asking for features without realising it already exists' :-p
Thanks @Incogdino for the investigation.
